apps/frontend/src/components/ApproveRedirect.svelte

```diff
@@ -5,6 +5,13 @@
   import SuccessIcon from './base/svg/SuccessIcon.svelte';
   import ErrorIcon from './base/svg/ErrorIcon.svelte';
   import Loader from './base/Loader.svelte';
+  // import { Circle } from 'svelte-loading-spinners';
```

Comment:
Lets remove commented out code. We generally don't want to merge commented code in